.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: rust
+sudo: false
+
+cache:
+  - cargo
+
+rust:
+  - stable
```

Comment:
It would be good to have nightly and beta in here too with `allow_failures` on for nightly as suggested in the [Travis CI Rust Guide](https://docs.travis-ci.com/user/languages/rust), but I don't mind doing that myself.

Reply:
Ok I'll do that...